tests/unit/src/main/scala/tests/TestMtagsResolver.scala

```diff
@@ -14,17 +14,12 @@ class TestMtagsResolver extends MtagsResolver {
 
   val default: MtagsResolver = MtagsResolver.default()
 
-  /**
-   * We don't need to check unsupported versions in tests and that makes the tests run longer.
-   */
   override def resolve(scalaVersion: String): Option[MtagsBinaries] = {
-    if (ScalaVersions.isSupportedAtReleaseMomentScalaVersion(scalaVersion))
-      default.resolve(scalaVersion).orElse(Some(MtagsBinaries.BuildIn))
-    else None
-  }
-
-  override def isSupportedScalaVersion(version: String): Boolean = {
-    ScalaVersions.isSupportedAtReleaseMomentScalaVersion(version)
+    default.resolve(scalaVersion).orElse {
+      if (ScalaVersions.isSupportedAtReleaseMomentScalaVersion(scalaVersion))
+        Some(MtagsBinaries.BuildIn)
```

Comment:
Would this not resolve always latest Scala 2.13 mtags?

Reply:
Only if `resolve` failed and we want to use supported Scala version. I think its used only in unit tests, where we don't publish mtags

Reply:
ach, right!